flora.cabal

```diff
@@ -103,27 +103,27 @@ library
     , envparse                   ^>=0.5
     , hashable                   ^>=1.3
     , http-types                 ^>=0.12
-    , lucid                      ^>=2.10
+    , lucid                      
     , lucid-alpine               ^>=0.1
     , lucid-svg                  ^>=0.7
     , mtl                        ^>=2.2
-    , optics-core                ^>=0.4
+    , optics-core                
     , optparse-applicative       ^>=0.16
     , password                   ^>=3.0
     , password-types             ^>=1.0
-    , pcre2                      ^>=2.0
+    , pcre2                      
     , pg-entity                  ^>=0.0
     , pg-transact                ^>=0.3
     , postgresql-simple          ^>=0.6
     , pretty                     ^>=1.1
-    , prometheus-client          ^>=1.1
-    , prometheus-metrics-ghc     ^>=1.0
-    , prometheus-proc            ^>=0.1
-    , PyF                        ^>=0.10
+    , prometheus-client          
+    , prometheus-metrics-ghc     
+    , prometheus-proc            
+    , PyF                        
     , raven-haskell              ^>=0.1
     , resource-pool              ^>=0.2
     , servant                    ^>=0.18
-    , servant-lucid              ^>=0.9
```

Comment:
I hope this isn't a problem, deleting bounds is usually a good strategy to make cabal happier, since now you don't need to align nix state of world with that of cabal.